resalloc_ibm_cloud/argparsers.py

```diff
@@ -127,3 +148,82 @@ def list_vms_parser():
 def list_deleting_volumes_parser():
     """ parser for listing vms """
     return _default_arg_parser(prog=_pfx("list-deleting-volumes"))
+
+
+def powervs_arg_parser():
+    """
+    Parser for the resalloc-ibm-cloud-powervs-vm utility.
+    """
+    parser = _default_arg_parser(prog=_pfx("powervs-vm"))
+
+    subparsers = parser.add_subparsers(dest="subparser")
+    subparsers.required = True
+    
+    parser_create = subparsers.add_parser(
+        "create", help="Create a PowerVS instance in IBM Cloud"
+    )
+    _add_common_create_args(parser_create)
+    parser_create.add_argument(
+        "--ssh-key-name", required=True, help="Name of the SSH key to use for the instance"
+    )
+    parser_create.add_argument(
+        "--processors", type=float, required=True, help="Number of processors (CPU cores)"
+    )
+    parser_create.add_argument(
+        "--processor-type",
+        choices=["dedicated", "shared", "capped"], 
+        help="Processor type (dedicated, shared, or capped)",
+        default="shared",
+    )
+    parser_create.add_argument(
+        "--ram", type=float, required=True, help="RAM memory in GB"
+    )
+    parser_create.add_argument(
+        "--network-id", type=str, nargs="+", help="ID of the network to attach to the instance"
+    )
+    parser_create.add_argument(
+        "--storage-type",
+        choices=["tier1", "tier3"],
+        default="tier3",
+        help="Storage tier type (tier1 for SSD, tier3 for HDD)",
+    )
+    parser_create.add_argument(
+        "--volumes",
+        type=str,
+        nargs="+",
+        help="Additional volumes to attach in format 'name:size_gb:type'"
+    )
+    _add_tags_argument(parser_create)
+    parser_create.add_argument(
+        "--system-type",
+        choices=["s922", "e980"],
+        help="Machine type (s922, e980)",
+        default="s922",
+    )
+    
+    parser_delete = subparsers.add_parser(
+        "delete", help="Delete PowerVS instance by its name from IBM Cloud"
+    )
+    parser_delete.add_argument("name")
+    
+    parser_list = subparsers.add_parser(
+        "list", help="List all PowerVS instances in IBM Cloud"
+    )
+    
+    return parser
```

Comment:
It is true, that this adds many more parameters compared to the original non-powerful builders. 

But to me, it seems that they are not new or different. It's just that for normal builders, we have them in a config file. I don't know where the config is on Fedora Copr but in case of Internal Copr, it is the deployment repository and `roles/copr/backend/templates/ibm-cloud-config.j2`

Can we unify the configuration and use either parameters or a config file for both types of builders or is there some reason why we can't/shouldn't?